demo/vl-template.html

```diff
@@ -1,52 +1,169 @@
 <!DOCTYPE html>
 <html>
+
 <head>
   <title>vl-template voorbeeld</title>
   <meta name="viewport" content="width=device-width">
   <script type="module" src="../vl-template.src.js"></script>
+  <script type="module" src="../node_modules/vl-ui-header/vl-header.js"></script>
+  <script type="module" src="../node_modules/vl-ui-footer/vl-footer.js"></script>
   <script type="module" src="../node_modules/vl-ui-grid/vl-grid.js"></script>
+  <script type="module" src="../node_modules/vl-ui-content-header/vl-content-header.js"></script>
+  <script type="module" src="../node_modules/vl-ui-image/vl-image.js"></script>
+  <script type="module" src="../node_modules/vl-ui-titles/vl-titles.js"></script>
+  <script type="module" src="../node_modules/vl-ui-typography/vl-typography.js"></script>
   <script type="module" src="demo.js"></script>
-  <link rel="stylesheet" href="custom.css"/>
+  <script type="module" src="custom.js"></script>
 </head>
 
 <body>
-<h1>vl-template demo</h1>
-<div>
-  <h2>Template</h2>
-  <pre>
-      <code class="line-numbers language-markup">
-
-      </code>
-    </pre>
   <vl-template>
-    <div slot="header"><p>header content</p></div>
+    <vl-header slot="header" data-vl-identifier="59188ff6-662b-45b9-b23a-964ad48c2bfb"></vl-header>
```

Comment:
feit da ge hier header en footer bij in den template steekt is "ter illustratie"? want niet verplicht om het gebruik van vl-template aan te tonen vermoed ik?

Reply:
Ter illustratie, inderdaad niet verplicht.